### PR TITLE
EDU-230 Remove default document from landing page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-230

Due to the fact that `global-alerts.js` was exposing both the helper method `getGlobalAlerts` and the hook `useGlobalAlerts` and due to the need of the extra dependency on `settings`, I decided to unify in the direction of using the hook, reason why I changed all pages that did not use the hook into functional components.

**Update**
Changed logic to display this alert banner on fullscreen pages as well, thus displaying it on all pages.

### CODE REVIEW

**Comment by laurentiu-ilici** [comment]
I don't see the banner on the home page, it seems to work on other pages, is this intentional? The story says "all pages"

**Comment by adrianaluchian** [comment]
> I don't see the banner on the home page, it seems to work on other pages, is this intentional? The story says "all pages"
There are no banners on the home page of any kind, all our banners are on all other pages.

**Comment by ahelmberger** [comment]
> > I don't see the banner on the home page, it seems to work on other pages, is this intentional? The story says "all pages"
> > There are no banners on the home page of any kind, all our banners are on all other pages.

It's just important to not forget to make the home page show the banner, because that's the most important place we want to have it. We have to adjust the page template in the consuming systems (maybe it would make sense to also change it in the test app in this repo?

**Comment by adrianaluchian** [comment]
> > > I don't see the banner on the home page, it seems to work on other pages, is this intentional? The story says "all pages"
> > > There are no banners on the home page of any kind, all our banners are on all other pages.
> 
> It's just important to not forget to make the home page show the banner, because that's the most important place we want to have it. We have to adjust the page template in the consuming systems (maybe it would make sense to also change it in the test app in this repo?

Ok my understanding was that we need to add this banner the same way as the other global banner 😬 
In this case I will decline the PR, make the changes and reopen.

**Comment by adrianaluchian** [comment]
Closing due to pending changes.

**Comment by laurentiu-ilici** [comment]
> > > I don't see the banner on the home page, it seems to work on other pages, is this intentional? The story says "all pages"
> > > There are no banners on the home page of any kind, all our banners are on all other pages.
> 
> It's just important to not forget to make the home page show the banner, because that's the most important place we want to have it. We have to adjust the page template in the consuming systems (maybe it would make sense to also change it in the test app in this repo?

Okay, then there are 3 others (which probably don't make a lot of sense to have the banner),  but I'll ask: login page, registration form page and password forgot.

**Comment by ahelmberger** [comment]
> > > > I don't see the banner on the home page, it seems to work on other pages, is this intentional? The story says "all pages"
> > > > There are no banners on the home page of any kind, all our banners are on all other pages.
> > 
> > 
> > It's just important to not forget to make the home page show the banner, because that's the most important place we want to have it. We have to adjust the page template in the consuming systems (maybe it would make sense to also change it in the test app in this repo?
> 
> Okay, then there are 3 others (which probably don't make a lot of sense to have the banner), but I'll ask: login page, registration form page and password forgot.

To be honest, I would just show it on **all** pages, basta :-)

File: src/components/default-page-template.js [hunk 1]
```diff
@@ -137,6 +137,25 @@ function DefaultPageTemplate({ children, fullScreen, headerActions, alerts }) {
     }
   ].filter(item => item.showWhen);
 
+  const renderAlert = alert => {
+    const shouldRenderAlert = !fullScreen || alert.showInFullScreen;
```
Comment (ahelmberger) [inline src/components/default-page-template.js:141]: Nice :+1:

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**